sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -64,8 +64,12 @@ def _to_microseconds(time_ns: int) -> int:
   return int(time_ns / _NANOSECOND_TO_MICROSECOND)
 
 
-class InferenceRunner(Generic[ExampleT, PredictionT, ModelT]):
-  """Implements running inferences for a framework."""
+class ModelHandler(Generic[ExampleT, PredictionT, ModelT]):
+  """Has the ability to load an ML model."""
```

Comment:
Maybe update this docstring?

Reply:
Similarly for the implementations, consider if the the docstrings need to be merged.